src/cdk/drag-drop/drag-ref.ts

```diff
@@ -33,6 +33,11 @@ import {getMutableClientRect, adjustClientRect} from './client-rect';
 import {ParentPositionTracker} from './parent-position-tracker';
 import {deepCloneNode} from './clone-node';
 
+// CSSStyleDeclaration does not include webkitTapHighlightcolor. Making a type for this to avoid
+// using `any` type.
+type _CSSStyleDeclaration = CSSStyleDeclaration & { webkitTapHighlightColor?: string|null};
```

Comment:
There's already a `DragCSSStyleDeclaration` interface inside `drag-styling.ts`. Maybe we should reuse it instead?

Reply:
Sure, that's sounds reasonable. I'm also okay with doing `CSSStyleDeclaration & { [key: string]: string|null|undefined }` if that's more convenient.

Reply:
Nah, I think it's better to have the stricter type checking.